Software/Development/SemesterProject/System/Include/scheduler.h

```diff
@@ -11,8 +11,7 @@
 #include "systemManager.h"
 #include <stddef.h>
 
-#define maxNumberTasks      10
-#define largestCyclicity    20000
+#define maxNumberTasks      5
 #define timerPeriodUs       50
```

Comment:
So conclusion is to do defines with capital letters

Reply:
Done!